README.md

```diff
@@ -47,35 +47,40 @@ ShapeShift's OSS 2nd generation Web application. (Under Development)
 
 ## Quick Start
 
-If you are using Linux and MacOS it works out of the box following the steps.<br/>
-⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL).
+If you are using Linux and MacOS it works out of the box following the steps.
 
-- Clone the repo
+> ⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL) and make sure to use the WSL filesystem to store your clone of the repository in order to avoid issues with line endings and enjoy better performance.
```

Comment:
`_Windows Subsystem for Linux_ (WSL)` should be a link?

Reply:
We could also mention in the same paragraph that no additional support will be provided for developers using WSL, because our focus is to support dev environments for Linux/Mac OS.

From what I read WSL2 is already a big improvement in an attempt to match a native Linux distro (using a VM for storage with an EXT4 file system). But obviously it's [unlikely to ever match 100%](https://learn.microsoft.com/en-us/windows/wsl/compare-versions) but they already claim "100% system call compatibility" apparently.

I do expect "hdwallet" which deals with WebUSB to be much more problematic with WSL (just reading the linked limitations above, but I have not tested it). My suggestion to explicitly limit the support would at least not discourage potential future contributors who just want to first test our code and experiment.

Reply:
Yeah, agree with where your head is at @gomesalexandre. Maybe to @firebomb1's suggestion we want to add language stating very clearly that we do not support approach this in any way, and so do not ask the engineering team about it - but if you want to play around with it good luck.

Reply:
Yeah, as long as we add a warning alongside the instructions, I'm fine with it @0xApotheosis

Reply:
Thanks, the requested changes have been applied. The Windows section has been moved at the end of quick start guide to also signify it isn't something we intend to support.